docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as Centos, AppVeyor, Circle and Travis. These servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
 
-[[ci-setup]]
+The following sections describe the various CI build servers Minishift utilizes.
+
+[[centos-ci]]
 == CentOS CI
 
-Minishift uses several CI build servers, including https://ci.centos.org/[CentOS CI]. The server
-builds incoming pull requests and pushes to master. It also archives build artifacts of successful
-builds.
+The https://ci.centos.org/[CentOS CI] server is used specifically to test integration tests, unit tests, code format tests and cross build tests in Linux (CentOS).
```

Comment:
Why "specifically". What's interesting with CentOS CI is that it is the only CI server which can run the integration tests, since it is the only one supporting nested virtualisation.
This entire part was based on input provided by @budhrg in Issue #426. Should we add this information here?
"cross build tests", not sure what you mean. There are not cross build tests. You maybe mix this up with cross compilation (aka generating the binaries for the other supported platforms as well)
Again, as per input provided in Issue #426, @budhrg did ask for further input but no information was forthcoming on this part.